src/main/java/com/atlassian/db/replica/api/DualConnection.java

```diff
@@ -44,7 +44,7 @@
  * Falls back to the main database if it cannot use a replica.
  */
 public final class DualConnection implements Connection {
-    public static final String CONNECTION_CLOSED_MESSAGE = "This connection has been closed.";
```

Comment:
Technically it removes part of API. Luckily we're still in 0.x.y SemVer.
Shall we go for 1.0.0 soon? We're gonna [use it on prod, which means it's high time](https://semver.org/#how-do-i-know-when-to-release-100).
We can still make breaking changes afterwards, just gotta communicate it via a major bump.

Reply:
It doesn't. The changes were not released ;)

> Shall we go for 1.0.0 soon? 

Anytime

Reply:
> The changes were not released ;)

Right!

Reply:
> Anytime

Right on, I'll release after #7 merge